app_dart/bin/generate_jspb.dart

```diff
@@ -66,6 +66,6 @@ Future<void> main(List<String> args) async {
   final YamlMap configYaml = loadYaml(configContent) as YamlMap;
   // There's an assumption that we're only generating builder configs from commits that
   // have already landed with validation. Otherwise, this will fail.
-  final pb.SchedulerConfig schedulerConfig = schedulerConfigFromYaml(configYaml);
+  final pb.SchedulerConfig schedulerConfig = await schedulerConfigFromYaml(configYaml, null);
```

Comment:
Seeing how this is used, I'd recommend instead of passing branch we add a parameter like

```
schedulerConfigFromYaml(pb.SchedulerConfig config, {
  bool ensureBringupTargets = true,
});
```

Then in the classes impacted we can ignore the check.

As it stands, it's hard to understand the reasoning in the API

Reply:
That's a very good point! Sorry for the delay, I guess I end up spending too much time on the shuttle bus commute today.

I have added the named parameter ensureBringupTargets. Eventually I kept the named parameter slug, because my understanding is that slug and the branch information it encodes are necessary for the githubFileContent function to work, and currently I am using githubFileContent to pull down tip of tree targets information.

I wasn't very sure if we can get schedulerConfigFromYaml to work with one named boolean parameter. Currently I am supplying two named parameters to the schedulerConfigFromYaml function. Maybe there is a way to pull information of tip of tree targets without supplying the branch information?